evm/src/main/java/org/hyperledger/besu/evm/operation/SDivOperation.java

```diff
@@ -71,10 +70,12 @@ public static OperationResult staticOperation(final MessageFrame frame) {
         resultBytes = resultBytes.slice(resultBytes.size() - 32, 32);
       }
 
-      final byte[] padding = new byte[32 - resultBytes.size()];
-      Arrays.fill(padding, result.signum() < 0 ? (byte) 0xFF : 0x00);
+      frame.pushStackItem(Bytes32.leftPad(resultBytes, result.signum() < 0 ? (byte) 0xFF : 0x00));
 
-      frame.pushStackItem(UInt256.fromBytes(Bytes.concatenate(Bytes.wrap(padding), resultBytes)));
+      //      final byte[] padding = new byte[32 - resultBytes.size()];
+      //      Arrays.fill(padding, result.signum() < 0 ? (byte) 0xFF : 0x00);
+      //
+      //      frame.pushStackItem(Bytes.concatenate(Bytes.wrap(padding), resultBytes));
```

Comment:
did you mean to leave this commented code?

Reply:
No, somehow it passed my sonar checks.  Will remove